src/feature/MeetTheHost.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import "./MeetTheHost.css";
+
+const MeetTheHost = () => {
+  return (
+    <div className="host-info-container">
+      <h2 className="header">Meet your Host</h2>
+      <div className="main-container">
+        <div className="leaf-side">
+          <div className="round-photo-component">
+            <img
+              className="raus-img"
+              src="https://a0.muscache.com/im/pictures/user/d62627ea-ea22-4cf1-b38a-152f1f86a9ed.jpg"
+              alt="RAUS"
+            />
+          </div>
+          <div className="verified-container">
+            <svg
+              version="1.1"
+              xmlns="http://www.w3.org/2000/svg"
+              width="32"
+              height="32"
+              viewBox="0 0 32 32"
+              role="img"
+              aria-label="check mark"
+              className="verifiedicon-svg"
+            >
+              <title>verified-sheild</title>
```

Comment:
Probably not all hosts are verified. We could use a prop here too that will tell us if the host is verified and only if the prop is `true` we display the shield.

Reply:
of course. I will do that. Also, as you mentioned in the next comment, this is a screenshot of my work:
![Screenshot 2024-09-20 175229](https://github.com/user-attachments/assets/ba5325b2-00a5-4fd3-9c7f-23d7024a61ac)

Reply:
Thanks for posting the screenshot! It looks great 👍 But you'll probably want to post it into the general conversation of the PR rather than here in this thread. Because I'm happy with the change you made with the shield, I'm going to click on "resolve conversation" and that will hide this thread and people will only be able to see it they click "Show resolved"